converter/value.go

```diff
@@ -48,4 +50,17 @@ type (
 		// json.Unmarshal.
 		Get(valuePtr ...interface{}) error
 	}
+
+	// RawValue is a representation of an unconverted, raw payload.
+	//
+	// This type can be used as a paramter or return type in workflows and activities to pass through
+	// a raw payload. Encoding/decoding of the payload is still done by the system.
+	RawValue struct {
```

Comment:
I wonder if we should consider `PayloadMarshaler` and `PayloadUnmarshaler` interfaces similar to `json.Marshaler` and `json.Unmarshaler` and just have `RawValue` implement both.

Reply:
If stack trace opt in, no prob. As for the interfaces, I think that's fairly uniquely Go, but ok if we don't want it.